dev/google-drive-files/README.md

```diff
@@ -0,0 +1,35 @@
+# Google Drive files
+
+https://drive.google.com/drive/folders/1a9ec9vzBV_D-AX9s_LOkBVy3ZXDC1kJT
+
+## Setup
+
+### gem install
+
+```console
+$ gem install google-apis-drive_v3
+```
+
+### Google service account
+
+1. Create a service account
+   * https://docs.cloud.google.com/iam/docs/service-accounts-create
+2. Create a key and download the JSON file
+   * https://docs.cloud.google.com/iam/docs/keys-create-delete#creating
+3. Set the path of the downloaded JSON file to the environment variable `GOOGLE_APPLICATION_CREDENTIALS`
+   * ```console
+     # Bash example
+     $ export GOOGLE_APPLICATION_CREDENTIALS=/path/to/service_account_json_key.json
+     ```
+
+## Generate file ids
+
+```console
+$ ./generate-file-ids.rb
+1rf8L-TygYD0FyNEFpG5yzvrz0aRzQWa_	OpenArm_v1.1_leader.STEP
```

Comment:
Can we also show folder name such as `STEP (assembly)/OpenArm_v1.1_leader.STEP`?